backend/fregepoc/indexers/models.py

```diff
@@ -7,6 +8,12 @@
 from django.utils.translation import gettext_lazy as _
 from github import Github
 
+from .sourceforge import SinglePageProjectsExtractor
```

Comment:
Could you make this imports absolute ? PEP 8 explicitly recommends using absolute imports. The same applies to the imports in the remaining files.